uservalidation/user.go

```diff
@@ -35,9 +35,9 @@ func Validate(user User, isSignUp bool) error {
 	return nil
 }
 
-func CheckName(name string) error {
-	re := regexp.MustCompile(`[^a-zA-ZäöüÄÖÜáéíóúÁÉÍÓÚàèìòùÀÈÌÒÙâêîôûÂÊÎÔÛãñõÃÑÕçÇ ^\p{L}'-]`)
+var re = regexp.MustCompile(`[^a-zA-ZäöüÄÖÜáéíóúÁÉÍÓÚàèìòùÀÈÌÒÙâêîôûÂÊÎÔÛãñõÃÑÕçÇ ^\p{L}'-]`)
 
+func CheckName(name string) error {
 	if name == "" {
```

Comment:
Tende a ser uma boa ideia sanitizar inputs string por espaços antes e depois do texto. É relativamente fácil em interfaces esquecer um espaço antes ou depois do valor de um campo e não perceber. `strings.TrimSpace()` tende a ser suficiente.
```suggestion
  sanitizedName := strings.TrimSpace(name)
  
	if sanitizedName == "" {
```